src/components/TextArea/TextArea.st.css

```diff
@@ -0,0 +1,83 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: applyUnit, applyOpacity, font, fallback, color, zeroAsTrue;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultBorderColor: color-5;
+    DefaultBackgroundColor: color-1;
+    DefaultTextColor: color-5;
+    DefaultPlaceholderColor: color-3;
+    DefaultFocusFontColor: color-1;
+    DefaultDisabledBorderColor: color-1;
+    DefaultFocusFontColor: color-1;
+    DefaultFont: "{theme: 'Body-M', size: '16px', lineHeight: '1.5em'}";
+
+    /*Defaults params goes here*/
+}
+
+:vars {
+    /*Overrides*/
+    BorderColor: --overridable;
+    BackgroundColor: --overridable;
+    TextColor: --overidable;
+    PlaceholderColor: --overidable;
+    FontPreference: --overidable;
```

Comment:
a picky comment: I'm not sure `FontPreference` is an understandable prop name.
We can maybe call it `TextFont`.